### PR TITLE
restic: add page

### PR DESCRIPTION
**Pull request by hdhauk**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by agnivade** [comment]
@hdhauk - Can you do a force push and remove the older commits ? There seems to be a merge commit at the end. If you just get the latest from master and apply just the new commit and then force push, I think that should do it.

**Comment by hdhauk** [comment]
@agnivade Looks better now?

File: pages/common/restic.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- Show backup snapshots currently stored in the repository:
+
+`restic -r {{/path/to/repository/}} snapshots`
+
+- Restore a specific backup snapshot to a target directory:
+
+`restic -r {{/path/to/repository/}} restore {{snapshot_id}} {{/path/to/target}}`
+
+- Clean up repository and keep only the most recent snapshot of each unique backup (same host and directory):
```
Comment (agnivade) [inline pages/common/restic.md]: Do we need to mention "same host and directory" ? I thought that was implied.

Comment (hdhauk) [reply pages/common/restic.md]: There are no default interval when snapshots are taken, they are taken whenever you run the backup command. For regular automatic backups you'll need to use cron or something.

You're right, its probably redundant – removed!

**Comment by pxgamer** [comment]
I've merged this as the grammatical change is pretty minor. 👍

**Pull request merged**